arch/risc-v/src/common/espressif/Kconfig

```diff
@@ -408,6 +418,33 @@ config ESPRESSIF_FLASH_MODE_QOUT
 
 endchoice # ESPRESSIF_FLASH_MODE
 
+if ESPRESSIF_SPIFLASH
+comment "General storage MTD configuration"
+
+config ESPRESSIF_MTD
+	bool "MTD driver"
+	default y
+	select MTD
+	select MTD_BYTE_WRITE
+	select MTD_PARTITION
+	---help---
+		Initialize an MTD driver for the SPI Flash, which will
+		add an entry at /dev for application access from userspace.
+
+config ESPRESSIF_SPIFLASH_MTD_BLKSIZE
+	int "Storage MTD block size"
+	default 64
+	depends on ESPRESSIF_MTD
+
```

Comment:
Please include a help for this parameter as well. What are the valid block sizes for ESP SPIFLASHes? Should it be a range?